sklearn/inspection/_partial_dependence.py

```diff
@@ -138,7 +139,9 @@ def _partial_dependence_recursion(est, grid, features):
     return averaged_predictions
 
 
-def _partial_dependence_brute(est, grid, features, X, response_method):
+def _partial_dependence_brute(
+    est, grid, features, X, response_method, sample_weight=None
+):
```

Comment:
Could you add a docstring?

Reply:
Thanks for the review @lorentzenchr!
The change I made to the `_partial_dependence_brute` function was relatively minor (I only added the `sample_weight` parameter), but I agree that it (and its pair `_partial_dependence_recursion`) could use more descriptive documentation commenting out the parameters and returns.

I think it's okay to take advantage of this PR to do this!
I added it in my last commit, blending some of the docstrings of `partial_dependence` with the peculiarities of each.